cmds/core/io/io.go

```diff
@@ -36,29 +36,24 @@ import (
 	"github.com/u-root/u-root/pkg/memio"
 )
 
+type cmdFunc func(addr int64, data memio.UintN) error
 type cmd struct {
-	f                 func(addr int64, data memio.UintN) error
+	f                 cmdFunc
 	addrBits, valBits int
 }
 
 var (
-	readCmds = map[string]cmd{
```

Comment:
Plan9 has no way to read/write to physical memory?

Reply:
no, it's a security hole and was hence never allowed. The IOport thing is a bit of a hole too but we've let it go.